_docs/howto/deploy/change-gcp-deployment.md

```diff
@@ -4,17 +4,27 @@ order: 104
 ---
 #  {{ page.title }}
 
-Note: If you used the Forseti setup wizard to deploy, your `deploy-forseti.yaml` 
-will have a timestamp suffix, e.g. `deploy-forseti-20171001000000.yaml`.
+This page describes how to change or _upgrade_ an existing installation of Forseti.
+
+There are four steps steps to upgrading existing deployments of Forseti.
+
+ 1. [Upgrade the codebase](#get-the-latest-version)
+ 1. [Check for differences in the deployment](#check-for-changes-to-the-deployment-template)
+ 1. [Check for and enable any new capabilities](#enable-new-features)
+ 1. [Upgrade the deployment](update-the-deployment)
```

Comment:
should this be (#update-the-deployment) instead?

Reply:
Yes, fixed!